### ISSUE TITLE
Windows 7 crash

### ISSUE THREAD

**Issue by trackfour** [Windower/Issues#693]
Getting crashes after pressing the final "play" button from POL. Sometime it will load and get stuck on the "connecting to lobby" message. Here is the latest dump file.

https://drive.google.com/file/d/0B1dp4msYQJbgeWlJUGRvNUFvMTQ/view?usp=sharing

**Comment by trackfour** [comment]
https://drive.google.com/file/d/0B1dp4msYQJbgLXRpOTdDS1h4Nms/view?usp=sharing

**Comment by trackfour** [comment]
Couple more dumps.
https://drive.google.com/file/d/0B1dp4msYQJbgczRXdFZrWGVfVHc/view?usp=sharing

**Comment by z16** [comment]
Does it happen with no plugins/addons loaded? To try that, start POL through Windower, then, before clicking "Play", go to Windower/scripts/autoload/autoload.txt and remove everything in that file so it's just a blank file. Then click on play and see if it happens still.

**Comment by trackfour** [comment]
Yep - it still happens. I had tried before with no plugins/addons loaded but I tried editing that file and it still happens. Without windower ffxi runs smooth, no problems. I thought maybe it was a problem with my install so I uninstalled and reinstalled everything and it still doesn't work. Here is the dump from following your instructions:
https://drive.google.com/file/d/0B1dp4msYQJbgblpkQm1YczF4VTg/view?usp=sharing

**Comment by z16** [comment]
Just to make sure it wasn't a typo, you're really on Windows 7? Because this is almost the perfect definition of the standard Windows 8 crash.

**Comment by trackfour** [comment]
Yep - windows 7. I thought that too when reading other crash issues.

**Comment by trackfour** [comment]
Weird thing is, it used to work just fine. I stopped playing and hadn't logged in for almost 2 years then just reactivated and ran into this issue. When the crash happens it lags out my cpu so hard it keeps crashing my whole system too if I can't get to task manager and get it shut down.

**Comment by z16** [comment]
Unfortunately that means we don't know what is causing this... every crash dump you provided gives different error information, which could point to heap corruption, but if so, is extremely hard to track down. I'll see what we can do, but I wouldn't hold my breath if I was you. You could try some of the things that people in the Windows 8 thread reported worked for them (downgrading drivers for example), and pray that one of these works for you as well.

**Comment by trackfour** [comment]
No problem - thanks for trying to help. If it was a driver issue I probably wouldn't be able to log in normally either, right?

**Comment by z16** [comment]
Some people reported it fixed that for them, and FFXI itself worked fine before as well. It's a really weird issue, and it usually points to something DirectX related, which Windower does have a hand in, so it's not out of the question.

**Comment by trackfour** [comment]
A lot of the time it doesn't crash immediately and I can click the "accept" button when ffxi starts up then it freezes up on "connecting to lobby server". It looks like others with windows 7 have the same issue. Thanks again, I will mess around and hopefully get it to work. If I figure it out I will let you know.

**Comment by trackfour** [comment]
I think I have discovered the problem in my case... I tried logging in on my laptop and ran into the same problem. The only thing I can recall that was different last time I was using FFXI was my antivirus software - Bitdefender free edition. I uninstalled Bitdefender and was able to login through windower with no problems. When searching for "bitdefender ffxi" I ran into an old thread from the POL forums that said that others ran into the same problem with Bitdefender software. I think Bitdefender has some conflicts with the way FFXI runs. Just FYI if people run into the same issues. I am going to try the same things on my desktop when I get home but will try to add some exceptions to Bitdefender first to see if I can get them both to get along without having to uninstall.
